ci/bazel-scripts/diff.sh

```diff
@@ -18,7 +18,7 @@ MERGE_BASE="${MERGE_BASE_SHA:-HEAD}"
 COMMIT_RANGE="$MERGE_BASE..${BRANCH_HEAD_SHA:-}"
 DIFF_FILES=$(git diff --name-only "${COMMIT_RANGE}")
 
-if grep -qE "(.*\.bazel|.*\.bzl|\.bazelrc|\.bazelversion)" <<<"$DIFF_FILES"; then
+if grep -qE "(.*\.bazel|.*\.bzl|\.bazelrc|\.bazelversion|mainnet-canisters\.json)" <<<"$DIFF_FILES"; then
```

Comment:
Should it also retrigger when `testnet/mainnet-canisters.json` was changed? If that's the case I would add a `*` in front.

Reply:
`testnet/mainnet-canisters.json` doesn't exist. You probably meant [testnet/mainnet_revisions.json](https://sourcegraph.com/github.com/dfinity/ic/-/blob/testnet/mainnet_revisions.json). I think that's not necessary since the bazel targets that depend on that file seem to have it listed as a dependency.